src/libsync/discovery.cpp

```diff
@@ -1178,16 +1167,19 @@ void ProcessDirectoryJob::processFileAnalyzeLocalInfo(
             item->_status = SyncFileItem::Status::NormalError;
         }
 
-        if (const auto folderQuota = folderQuotaAvailable(item);item->_size > folderQuota && folderQuota > -1) {
-            qCDebug(lcDisco) << "Folder" << item->_file
-                             << "item used:" << item->_folderQuota.bytesUsed
-                             << "item available:" << item->_folderQuota.bytesAvailable
-                             << "- quota used: " << serverEntry.folderQuota.bytesUsed
-                             << "- quota available: " << serverEntry.folderQuota.bytesAvailable;
+        const auto isServerEntryValid = serverEntry.isValid();
+        if (const auto folderQuota = folderBytesAvailable(item, isServerEntryValid);
+            item->_size > folderQuota
+            && folderQuota > -1) {
+
+            qCDebug(lcDisco) << "Quota exceeded for item:" << item->_file
```

Comment:
should we change the level ?
debug level means that it will be missing from production logs